src/utils/haptics.ts

```diff
@@ -13,7 +13,7 @@ export const HapticFeedbackTypes = {
   selection: 'selection',
 };
 
-const hapticToTrigger = haptic => ({
+const hapticToTrigger = (haptic: any) => ({
```

Comment:
This has type `any` but I think it should be `keyof typeof HapticFeedbackTypes` if possible, since that is what is used in the definition of `haptics` using this function.